projects/packages/connection/src/class-rest-connector.php

```diff
@@ -521,4 +544,51 @@ public function connection_authorize_url( $request ) {
 		);
 	}
 
+	/**
+	 * The endpoint tried to partially or fully reconnect the website to WP.com.
+	 *
+	 * @since 9.9.0
+	 *
+	 * @param \WP_REST_Request $request The request sent to the WP REST API.
+	 *
+	 * @return \WP_REST_Response|WP_Error
+	 */
+	public static function update_user_token( $request ) {
```

Comment:
Should we add an extra check if the given user with id `$token_parts[2]` exists? Just to be on the safe side, as I noticed that `Tokens::update_user_token` doesn't check either...

Reply:
Good idea, done: https://github.com/Automattic/jetpack/pull/19994/commits/d11b21b27642f843ff3bbf062b2558575bc2a9e0